### PR TITLE
Fix Infinity Wallet Bug on Docs Build

### PR DESCRIPTION
**Pull request by leightkt**
### Description
Remove @web3-react/types from vite external rollups in doc svelte.config.js and Re-add infinity wallet to docs

To Test:
cd docs
yarn && yarn build
yarn run preview
connect inifinity wallet (and try a few others too for good measure)

expect wallet connection successful and no errors in the console. 

### Checklist
- [ ] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [ ] Check the box that allows repo maintainers to update this PR
- [ ] Test locally to make sure this feature/fix works
- [ ] Run `yarn check-all` to confirm there are not any associated errors
- [ ] Confirm this PR passes Circle CI checks
- [ ] Add or update relevant information in the documentation

### Docs Checklist
- [ ] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [ ] Add/update the appropriate package README (if applicable)
- [ ] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [ ] Add/update the related package in the `docs/package.json` file (if applicable)

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

#### Tests with demo app (injected)
- [ ] send transaction
- [ ] switch chains
- [ ] sign message
- [ ] sign typed message
- [ ] disconnect

#### Tests with demo app (SDK)
- [ ] send transaction
- [ ] switch chains
- [ ] sign message
- [ ] sign typed message
- [ ] disconnect

### CODE REVIEW

File: docs/svelte.config.js [hunk 1]
```diff
@@ -39,7 +39,6 @@ const config = {
             '@web3-react/metamask',
             '@web3-react/network',
             '@web3-react/walletconnect',
-            '@web3-react/types',
```
Comment (Adamj1232) [inline docs/svelte.config.js:42]: @leightkt have you confirmed all other wallets thats use `@web3-react/types` still work within the docs?
I dont know which wallets this was added here for but was added to this list for some wallet

Comment (leightkt) [reply docs/svelte.config.js:42]: @Adamj1232 Looking in the yarn.lock file- it looks like only @infinitywallet/infinity-connector and @web3-react/abstract-connector (an @infinitywallet/infinity-connector) use @web3-react/types

Comment (Adamj1232) [reply docs/svelte.config.js:42]: Hmm strange as that was added prior to the infinity wallet 🤔 may be an outdated dep
Were you able to test walletConnect by chance? I see they are a web3-react package also.
Can we remove any of the other web3-react deps in there?

**Review by Adamj1232 — APPROVED** [review]

File: docs/svelte.config.js [hunk 2]
```diff
@@ -33,16 +33,6 @@ const config = {
...
         rollupOptions: {
-          external: [
-            '@web3-react/core',
-            '@web3-react/eip1193',
-            '@web3-react/metamask',
-            '@web3-react/network',
-            '@web3-react/walletconnect',
-            '@web3-react/types',
-            '@web3-react/url',
-            '@web3-onboard/*'
```
Comment (Adamj1232) [inline docs/svelte.config.js:44]: I was wrong, we still want the `web3-onboard/*` within the external list. My bad!

Comment (leightkt) [reply docs/svelte.config.js:44]: no prob- readded!

**Pull request merged**